prompb/rpc.pb.gw.go

```diff
@@ -45,6 +45,23 @@ func request_Admin_TSDBSnapshot_0(ctx context.Context, marshaler runtime.Marshal
 
 }
 
+var (
```

Comment:
There's no need to update all this, v1 is where it's at.

Reply:
No Need to update V2 apis?

Reply:
No need.